src/utils/assets.js

```diff
@@ -37,6 +37,10 @@ export const transformAssetsToObject = (assetsArray: Asset[] = []): Assets => {
   }, {});
 };
 
+export const getAssetsAsList = (assetsObject: Assets = {}): Asset[] => {
```

Comment:
Why is this argument optional? It looks like every place already used directly `Object.keys(assets)` without checking anything, can't we just do the same here?

Reply:
Sure thing!